.editorconfig

```diff
@@ -4,13 +4,19 @@
 root = true
 
 [*]
-indent_style = space
```

Comment:
I'm not sure I'm down with turning off spaces. Is there a prettier default for spaces? I'm also fine with spaces=2 on ts/tsx files.

Reply:
i'm not a fan of using tabs at all and always prefer spaces. i'm also not a fan of just letting some system define what defaults are, which is why there was an .editorconfig to begin with.

Reply:
here's the default config for js: https://github.com/prettier/prettier/blob/main/src/language-js/options.js
im ok with forcing space chars, none of the supported prettier languages use tabs

Reply:
> im ok with forcing space chars, none of the supported prettier languages use tabs

ok, revert/add that back in please.

maybe the other stuff is good?

Reply:
oh, i see you did. thanks

Reply:
already did, check blow